options/locale/locale_en-US.ini

```diff
@@ -498,7 +500,7 @@ migrate.clone_local_path = or local server path
 migrate.permission_denied = You are not allowed to import local repositories.
 migrate.invalid_local_path = Invalid local path, it does not exist or not a directory.
 migrate.failed = Migration failed: %v
-migrate.lfs_mirror_unsupported = Mirroring LFS objects is not supported - use 'git lfs fetch --all' and 'git lfs push --all' instead.
+migrate.lfs_mirror_unsupported = Mirroring LFS objects is not supported - use 'git lfs fetch --all' and 'git lfs push --all' instead.
```

Comment:
What changed here?

Reply:
It was a encoding issue caused by my IDE.

Reply:
Oh, can you please revert it?

Reply:
Reverted it with https://github.com/go-gitea/gitea/pull/2211/commits/3142af370288594ba9b2907bfee58f694eb202a8